csaf/parser.py

```diff
@@ -165,12 +165,15 @@ def _process_vulnerabilities(self):
             if "discovery_date" in vulnerability:
                 vuln_info.set_value("discovery_date", vulnerability["discovery_date"])
             if "flags" in vulnerability:
+                products = []
                 for flag in vulnerability["flags"]:
                     if "label" in flag:
                         vuln_info.set_value("justification", flag["label"])
-                    vuln_info.set_value("created", flag["date"])
+                    if "date" in flag:
+                        vuln_info.set_value("created", flag["date"])
                     for product in flag["product_ids"]:
-                        vuln_info.set_value("Product", product)
+                        products.append(product)
+                    vuln_info.set_value("Product", products)
```

Comment:
shouldn't it be
```suggestion
                    vuln_info.set_value("Products", products)
```
I noticed that all names are lower cased, except of `Product`, @anthonyharrison  is that a bug or feature?
to make it consistent I would suggest:
```suggestion
                    vuln_info.set_value("products", products)
```